documentation/modules/ROOT/pages/connectors/spanner.adoc

```diff
@@ -0,0 +1,1201 @@
+// Category: debezium-using
+// Type: assembly
+[id="debezium-connector-for-spanner"]
+= {prodname} connector for Cloud Spanner
+:context: Cloud Spanner
+:mbean-name: {context}
+:toc:
+:toc-placement: macro
+:linkattrs:
+:icons: font
+:source-highlighter: highlight.js
+
+toc::[]
+
+{prodname}'s Cloud Spanner connector consumes and outputs Cloud Spanner change streams data into a Kafka topic.
+
+A Cloud Spanner link:https://cloud.google.com/spanner/docs/change-streams[change stream] watches and streams out a Cloud Spanner database's data changes - inserts, updates, deletes - in near-real-time. The Cloud Spanner connector abstracts away the details of querying the Cloud Spanner change streams. With this connector, you don't have to manage the change streams partition lifecycle, which is necessary when you link:https://cloud.devsite.corp.google.com/spanner/docs/change-streams/details#query[use the Spanner API directly].
+
+The connector does not support the snapshot feature at the moment. The first time it connects to a Cloud Spanner database, it will start streaming changes from either the user-provided start timestamp or the current time, if the user did not provide a start timestamp.
```

Comment:
I mentioned this, because the other Debezium documentation (Vitess) mentioned the same sentence, and I'm trying to keep close to the structure.

Agreed with the next sentence.